src/Model/BaseManager.php

```diff
@@ -80,7 +80,7 @@ public function findBy(array $criteria, array $orderBy = null, $limit = null, $o
 
     public function findOneBy(array $criteria, array $orderBy = null)
     {
-        return $this->getRepository()->findOneBy($criteria, $orderBy);
+        return $this->getRepository()->findOneBy($criteria);
```

Comment:
Was this a valid call in old symfony version?

Reply:
I think it was never valid, at least I don't see it in [older versions of ObjectRepository](https://github.com/doctrine/persistence/commits/v1.0.0/lib/Doctrine/Common/Persistence/ObjectRepository.php).